install/helm/agones/templates/crds/k8s/_io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta.yaml

```diff
@@ -16,37 +17,27 @@
 
 {{- define "io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta" }}
 description: ObjectMeta is metadata that all persisted resources must have, which includes all objects users must create.
-type: object
 properties:
   annotations:
-    description: 'Annotations is an unstructured key value map stored with a resource that may be set by external tools to store and retrieve arbitrary metadata. They are not queryable and should be preserved when modifying objects. More info: http://kubernetes.io/docs/user-guide/annotations'
-    type: object
     additionalProperties:
       type: string
+    description: "Annotations is an unstructured key value map stored with a resource that may be set by external tools to store and retrieve arbitrary metadata. They are not queryable and should be preserved when modifying objects. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations"
+    type: object
   creationTimestamp:
-    description: |-
-      CreationTimestamp is a timestamp representing the server time when this object was created. It is not guaranteed to be set in happens-before order across separate operations. Clients may not set this value. It is represented in RFC3339 form and is in UTC.
-
-      Populated by the system. Read-only. Null for lists. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
-    type: string
     format: date-time
-    nullable: true
+    type: string
```

Comment:
Looks like there might be a bug in `makeCreationTimestampNullable` - since this field isn't nullable.

Reply:
`makeCreationTimestampNullable` code modifies the "creationTimestamp" field under "properties" nullable to true in `io.k8s.apimachinery.pkg.apis.meta.v1.ObjectMeta.json`

```
    "creationTimestamp": {
      "$ref": "#/definitions/io.k8s.apimachinery.pkg.apis.meta.v1.Time",
      "description": "CreationTimestamp is a timestamp representing the server time when this object was created. It is not guaranteed to be set in happens-before order across separate operations. Clients may not set this value. It is represented in RFC3339 form and is in UTC.\n\nPopulated by the system. Read-only. Null for lists. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata",
      "nullable": true
    },
```

Reply:
But it's not in this file that I see here. It should be in here (unless tests pass?)

Reply:
`makeCreationTimestampNullable` didn't have bug I guess, All the changes are happing only in `tmp` dir. But the nullable=true is present in the `orig` dir if I am correct. Let me check..